src/Debugger-Tests/StepIntoTest.class.st

```diff
@@ -24,20 +24,6 @@ StepIntoTest >> stepA1 [
 	^42
 ]
 
-{ #category : #tests }
-StepIntoTest >> testDetectionOfExecutionEnd [
```

Comment:
This test does not make any sense because stepInto at the end of process block does not terminates it. Try to do it in the debugger on any doIt.
With correctly simulated process this test does not work. But it is the expected behaviour.